app/controllers/admin/take_part_pages_controller.rb

```diff
@@ -53,4 +55,20 @@ def take_part_page_params
       :title, :summary, :body, :image, :image_alt_text, :image_cache
     )
   end
+
+  def patch_getinvolved_page_links
+    get_involved_cid = "dbe329f1-359c-43f7-8944-580d4742aa91"
+    pages = []
+
+    TakePartPage.in_order.each do |page|
+      pages << page.content_id
+    end
```

Comment:
As @ChrisBAshton suggested on Slack, this can be done in one line using the `map` method.  This also means you won't need to initialise the `pages` array, as `map` always returns an array.
```suggestion
    pages = TakePartPage.in_order.map(&:content_id)
```